tests/content_api_test.py

```diff
@@ -574,3 +575,89 @@ def test_publish_item_with_attachments(self):
         with self.capi.test_client() as c:
             response = c.get(attachments[0]['href'], headers=self._auth_headers(subscriber))
             self.assertEqual(200, response.status_code, attachments[0]['href'])
+
+    def test_items_default_sorting(self):
+        subscriber = {'_id': 'sub1'}
+        headers = self._auth_headers(subscriber)
+
+        self.content_api.publish(
+            {'_id': 'aaaa', 'guid': 'aaa', 'urgency': '1', 'type': 'text'},
+            [subscriber]
+        )
+        time.sleep(1)
+        self.content_api.publish(
+            {'_id': 'bbbb', 'guid': 'bbb', 'urgency': '2', 'type': 'text'},
+            [subscriber]
+        )
+        time.sleep(1)
+        self.content_api.publish(
+            {'_id': 'cccc', 'guid': 'ccc', 'urgency': '3', 'type': 'text'},
+            [subscriber]
+        )
+        time.sleep(1)
+        self.content_api.publish(
+            {'_id': 'dddd', 'guid': 'ddd', 'urgency': '4', 'type': 'text'},
+            [subscriber]
+        )
+        time.sleep(1)
+        self.content_api.publish(
+            {'_id': 'eeee', 'guid': 'eee', 'urgency': '5', 'type': 'text'},
+            [subscriber]
+        )
+
+        with self.capi.test_client() as c:
+            # no filters
+            response = c.get('items', headers=headers)
+            self.assertEqual(200, response.status_code)
+            data = json.loads(response.data)
+            self.assertListEqual([i['urgency'] for i in data['_items']], ['5', '4', '3', '2', '1'])
+
+            # with filtering
+            response = c.get('items?type=["text"]', headers=headers)
+            self.assertEqual(200, response.status_code)
+            data = json.loads(response.data)
+            self.assertListEqual([i['urgency'] for i in data['_items']], ['5', '4', '3', '2', '1'])
+
+            # with filtering and custom sorting
+            response = c.get('items?type=["text"]&sort=[("versioncreated",-1)]', headers=headers)
+            self.assertEqual(200, response.status_code)
+            data = json.loads(response.data)
+            self.assertListEqual([i['urgency'] for i in data['_items']], ['5', '4', '3', '2', '1'])
```

Comment:
same here, the ordering doesn't change, it would be better to change order of initial items so we can check sorting actually worked.

Reply:
I will change it here to ascending.